backend/gene_info/api/v1.py

```diff
@@ -5,11 +5,20 @@
     NotFoundHTTPException,
     ForbiddenHTTPException,
 )
+from backend.gene_info.api.ensembl_ids import GeneChecker
 
 
-def gene_info(geneID: string):
+def gene_info(gene: string = "", geneID: string = ""):
```

Comment:
I would use None as the default and then check for that.
Also since it will ignore the gene name if the gene id is given its probably worth adding a note in case of a mismatch. And adding an explicit test case to show thats the expected behavior